spec/application.js

```diff
@@ -47,6 +47,7 @@ module.exports = {
     answers: Overwrite({ valueSchema: answersSchema }),
     comment: Overwrite({ valueSchema: commentSchema }),
     decision: Overwrite({ valueSchema: decisionSchema }),
+    profileId: Overwrite({ valueSchema: messageId }),
```

Comment:
should this be in `staticProps` I think that could be good, but maybe it doesn't matter.

Reply:
hmmm... yeah I think it should be in staticProps and `required: true` ... because if it's not there then you look at an application it will have absolutely no name on it....

but that means it's not really an `opts` and we're looking at a major version change.
If we're doing a major version change maybe we rip out that classic API which makes any type of profile?

Reply:
:fire: happy to discuss more

Reply:
so yeah make this a staticProp

Reply:
Added as a staticProp but i think we said we'd make it not required so it doesnt break older applications. Hope i remembered that right. I've made this change and changes the signature of the create method to:
`ssb.tribes.application.create(groupId, profileId, groupAdmins, opts, cb)` hope this is correct?

Reply:
yep you remembered right. I think this should work for old applications.
This is a breaking change to the API of application.create.

Possible solutions to avoid major version (which would be nice) :
- put profileId in opts (I think this is a good solution, because our schema says it is optional)
- test how many args there are when people use this. if `typeof opts === 'function'`, then assign the variables to the right things and set profileId (arg!)